includes/SpecialManageWikiExtensions.php

```diff
@@ -0,0 +1,213 @@
+<?php
+class SpecialManageWikiExtensions extends SpecialPage {
+	function __construct() {
+		parent::__construct( 'ManageWikiExtensions', 'managewiki-extensions' );
```

Comment:
you're adding a new 'managewiki-extensions' right?

Reply:
I tried keeping it "managewiki" but then the title page did not work correctly.

Reply:
the second param is user rights required to view the page, it should definitely be 'managewiki'.